GitSavvy.sublime-settings

```diff
@@ -233,5 +233,11 @@
         "rebase",
         "tags",
         "graph"
-    ]
+    ],
+
+    /*
+        Set this value to true if you'd like GitSavvy to prompt you for a
+        reset mode (e.g. --hard, --soft) on reset operations.
+     */
+    "prompt_for_reset_mode": false
```

Comment:
This seems like it'd be useful for the majority of users that are doing a reset, so let's turn it on by default.